CorsixTH/Lua/languages/german.lua

```diff
@@ -124,13 +122,52 @@ tooltip.staff_window.center_view = "Linksklick, um zur Person zu springen, Recht
 -- These strings are missing in some versions of TH (unpatched?)
 confirmation.restart_level = "Sind Sie sicher, dass Sie das aktuelle Level von vorne beginnen möchten?"
 
+
+
+-- reported as missing, although they are only in OVERRIDE section of english.lua
+level_progress.hospital_value_enough = "Halten Sie den Wert des Krankenhauses über %d und kümmern Sie sich um ihre anderen Probleme um das Level zu gewinnen."
+level_progress.cured_enough_patients = "Sie haben genug Patienten geheilt, aber die Qualität des Krankenhauses muss zunehmen um das Level zu gewinnen."
+
+--
+multiplayer.players_failed = "Der/die folgende(n) Spieler hat/haben das letzte Ziel nicht erreicht:"
+multiplayer.everyone_failed = "Alle haben das letzte Ziel nicht erreicht. Also können alle weiterspielen!"
+
+--
+disease_discovered_patient_choice.need_to_employ = "Stellen Sie eine(n) %s ein um diese Situation lösen zu können."
+
+
+-- remove invisible hyphens. unfortunately it is impossible to fix the wrong case, since
+-- the case determines the font
+policy = {
+  diag_termination = "diagnoseerstellung",
+  diag_procedure = "diagnoseverfahren",
+}
+
+-- remove hyphen since it makes it use the wrong font. again, the case cannot be fixed
+casebook = {
+  deaths = "ablebensfälle", --the ä looks weird as well
+}
+
+-- only drug_improved_1 is in OVERRIDE section of english.lua
+-- ..._1 is used to prevent 'the the squits' AKA orig [4,30], which is simply "Durchfall"
```

Comment:
So we do think this bug was only in English?

Reply:
It seems so, although there might be other languages that have unfortunate disease names. The thing is – it seems like most translators didn't know that the second string is a specific workaround. I've looked at several other files and they had different strings.